Source/relay/TourGuide/Quests/Level 11 - Desert.ash

```diff
@@ -175,6 +175,15 @@ void QLevel11DesertGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEnt
         else
             subentry.entries.listAppend("Use your desert sightseeing pamphlets. (+15% exploration)");
     }
+    int campgroundMilestoneCount = __campground[$item[milestone]];
+    if (campgroundMilestoneCount > 0) {
+        string milestonesPlural = campgroundMilestoneCount == 1 ? "" : "s";
```

Comment:
this is cute. i believe ezan has a way to do pluralization built in somewhere in our functions but i prefer this because i always forget the syntax anyway.